run_tests.sh

```diff
@@ -1 +1 @@
-rm -rf project/.test_* && py.test -s --ds=test_settings --verbose --cov ./unicoremc unicoremc "$@"
+rm -rf project/.test_* && py.test -s --ds=test_settings --verbose --cov ./unicoremc --cov ./organizations unicoremc organizations "$@"
```

Comment:
Updated. `run_tests.sh` is still around to delete old test directories. I'll create an issue for all tests to tear down correctly.